Polli/Reinicio.py

```diff
@@ -0,0 +1,115 @@
+from dataset_alunas import dataset
+
+dataset
+
+def main():
+    print("\n---  Seja bem vinda a Escola do Reprograma!  ---")
+    print("Sistema de informações de alunas")
+
+    while True:
+        cod_opcao = obter_opcao()
+        
+        if cod_opcao == 1:
+            incluir_nova_aluna()
+        elif cod_opcao == 2:
+            consultar_lista_alunas()
+        elif cod_opcao == 3:
+            consultar_faltas_aluna()
+        elif cod_opcao == 4:
+            consultar_notas_aluna()
+        elif cod_opcao == 5:
+            consultar_status_aprovacao()
+        elif cod_opcao == 6:
+            print("Encerrando o programa...")
+            break
+
+def obter_opcao():
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+    return codigo_opcao
+
+    
+def incluir_nova_aluna():
+    print("Insira os seguintes dados: ")
+    nome = str(input("Nome da aluna: ")) #Recebo nome da aluna
+    sobrenome = str(input("Sobrenome da aluna: ")) #Recebo sobrenome da aluna
+    turma = int(input("Turma da aluna (nº): ")) #Recebo nome a turma
+    lista_notas = (input('Digite as notas da aluna separadas por vírgula: '))
+    lista_presenca = bool(input('Digite True para a presença da aluna e False para a sua ausência: '))    
```

Comment:
Sobre utilizar o bool, o código está tentando converter a entrada do usuário para um valor booleano. Em Python, bool() converte uma string não vazia para True, independentemente do conteúdo da string. Por exemplo:
entrada = bool(input("Digite algo: "))

Se o usuário digitar qualquer coisa, como "True", "False", "Sim", "Não" ou qualquer palavra, bool() vai considerar isso como True. Aqui a sugestão é utilizar a estrutura que construimos no sábado.

**Sugestão:**
```suggestion
    lista_presenca = obter_presença()

def obter_presença():
    quantidade_aulas = int(input('Digite quantas aulas houveram: '))
    lista_presença = []

    for contador in range(quantidade_aulas):
        while True:
            entrada = input(f'Presença na aula #{contador +1} (True/False): ')
            if entrada in ['True', 'False']:
                lista_presença.append(eval(entrada))
                break
            else:
                print('Informação inválida. Insira apenas True ou False')

    return lista_presença
```